src/splitInteger.test.js

```diff
@@ -8,12 +8,23 @@ describe(`Function 'splitInteger':`, () => {
   });
 
   it(`should return an array`, () => {
+  });
 
+  it(`should return an array`, () => {
+    expect(splitInteger(32, 8)).toBeInstanceOf(Array);
   });
 
   it(`should return an array with 'num' if parts = 1`, () => {
+    expect(splitInteger(20, 1)).toMatchObject([20]);
```

Comment:
You can also use more simple ".toEqual([20]);" here. But this one works too

Reply:
ok